src/action/local.ts

```diff
@@ -0,0 +1,35 @@
+import {PathLike} from 'fs';
+import {Action} from '../action';
+import {Output} from '../config/output';
+import {SPACES_TO_INDENT_JSON} from '../json';
+import { Logger } from '../logging';
+
+export class Local implements Action {
```

Comment:
Don't expect users to run this locally, but is not important.

Reply:
This is mainly for development. Without this, its almost impossible to debug stuff. I spent weeks working on this component, I know what I am talking about ;-)